abtesting/Shared/AppConfig.swift

```diff
@@ -57,6 +57,31 @@ class AppConfig: ObservableObject {
     }
   }
 
+  @available(iOS 15, *)
+  func updateFromRemoteConfigAsync() async {
+    let remoteConfig = RemoteConfig.remoteConfig()
+    do {
+      let status = try await remoteConfig.fetch(withExpirationDuration: 0)
```

Comment:
Consider using the fetchAndActivate API instead of the separate fetch and activate.

Reply:
This will be a "larger" change (refactor both iOS 15 and previous portions, replace RemoteConfigFetchStatus extension with one for RemoteConfigFetchAndActivateStatus, remove activate `changed` functionality), so I'll make it the last commit in case we would rather stick with separate calls 👍

Reply:
Will be a separate PR